tests/test_data.py

```diff
@@ -426,6 +426,10 @@ def test_Kirk19_ingest(db):
     t = db.query(db.ProperMotions).filter(db.ProperMotions.c.reference == ref).astropy()
     assert len(t) == 182, f"found {len(t)} proper motion entries for {ref}"
 
+    # Test for Best18 proper motions added from Pan-STARRS catalog
+    ref = "Best18"
+    t = db.query(db.ProperMotions).filter(db.ProperMotions.c.reference == ref).astropy()
+    assert len(t) == 1966, f"found {len(t)} proper motion entries for {ref}"
```

Comment:
Move both Kirk19 and Best18 proper motion test to test_data_astrometry

Reply:
Work with Kasey to write a test of *adopted" proper motions for Best18, Best20, and GaiaEDR3

Reply:
Here's an example for parallaxes:
```python
t = (
        db.query(db.Parallaxes)
        .filter(and_(db.Parallaxes.c.reference == ref, db.Parallaxes.c.adopted == 1))
        .astropy()
    )
    assert (
        len(t) == 1077
    ), f"found {len(t)} adopted parallax reference entries for {ref}"